doc/ProfilesJSONDocumentation.md

```diff
@@ -0,0 +1,57 @@
+| `useAcrylic` | _Required_ | Boolean | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. |
+| `colorTable` | Optional | Array[String] | Array of colors used in the profile if `colorscheme` is not set. Colors use hex color format: `"#rrggbb"`. Ordering is as follows: `[black, red, green, yellow, blue, magenta, cyan, white, bright black, bright red, bright green, bright yellow, bright blue, bright magenta, bright cyan, bright white]` |
+| `cursorHeight` | Optional | Integer | Sets the height of the cursor. Only works when `cursorShape` is set to `"vintage"`. Accepts values from 25-100. |
+| `foreground` | Optional | String | Sets the foreground color of the profile. Overrides `foreground` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
+| `icon` | Optional | String | Image file location of the icon used in the profile. Displays within the tab and the dropdown menu. |
+| `scrollbarState` | Optional | String | Defines the visibility of the scrollbar. Possible values: `"visible"`, `"hidden"` |
+| **schemes** | | **Array[Object]** | **Properties listed below are specific to each color scheme.** |
+| `background` | _Required_ | String | Sets the background color of the color scheme. Uses hex color format: `"#rrggbb"`. |
+| `black` | _Required_ | String | Sets the color used as ANSI black. Uses hex color format: `"#rrggbb"`. |
+| `blue` | _Required_ | String | Sets the color used as ANSI blue. Uses hex color format: `"#rrggbb"`. |
+| `brightBlack` | _Required_ | String | Sets the color used as ANSI bright black. Uses hex color format: `"#rrggbb"`. |
+| `brightBlue` | _Required_ | String | Sets the color used as ANSI bright blue. Uses hex color format: `"#rrggbb"`. |
+| `brightCyan` | _Required_ | String | Sets the color used as ANSI bright cyan. Uses hex color format: `"#rrggbb"`. |
+| `brightGreen` | _Required_ | String | Sets the color used as ANSI bright green. Uses hex color format: `"#rrggbb"`. |
+| `brightPurple` | _Required_ | String | Sets the color used as ANSI bright purple. Uses hex color format: `"#rrggbb"`. |
+| `brightRed` | _Required_ | String | Sets the color used as ANSI bright red. Uses hex color format: `"#rrggbb"`. |
+| `brightWhite` | _Required_ | String | Sets the color used as ANSI bright white. Uses hex color format: `"#rrggbb"`. |
+| `brightYellow` | _Required_ | String | Sets the color used as ANSI bright yellow. Uses hex color format: `"#rrggbb"`. |
+| `cyan` | _Required_ | String | Sets the color used as ANSI cyan. Uses hex color format: `"#rrggbb"`. |
+| `foreground` | _Required_ | String | Sets the foreground color of the color scheme. Uses hex color format: `"#rrggbb"`. |
+| `green` | _Required_ | String | Sets the color used as ANSI green. Uses hex color format: `"#rrggbb"`. |
+| `name` | _Required_ | String | Name of the color scheme. |

```

Comment:
I understand that you're alphabetizing the parameters by category (required, then optional)... but it is really awkward to have the `name` property for schemes be buried down here (as opposed to at the top) and to have the `foreground` and `background` properties not next to each other in the list.

Reply:
I'm happy to rearrange based on property relevance?

Reply:
I'd probably do name, then fg/bg, then the rest of the colors